ChangeSet
  1.1525.2.4 04/04/26 16:37:38 petri.koistinen@iki.fi +1 -0
  [SUNRPC]: Missing NULL kmalloc check in unix_domain_find().

  net/sunrpc/svcauth_unix.c
    1.24 04/04/26 16:37:19 petri.koistinen@iki.fi +2 -0
    [SUNRPC]: Missing NULL kmalloc check in unix_domain_find().

.........................................................................
diff -Nru a/net/sunrpc/svcauth_unix.c b/net/sunrpc/svcauth_unix.c
--- a/net/sunrpc/svcauth_unix.c	Tue Apr 27 12:00:32 2004
+++ b/net/sunrpc/svcauth_unix.c	Tue Apr 27 12:00:32 2004
@@ -51,6 +51,8 @@
 		return rv;
 
 	new = kmalloc(sizeof(*new), GFP_KERNEL);
+	if (new == NULL)
+		return NULL;
 	cache_init(&new->h.h);
 	atomic_inc(&new->h.h.refcnt);
 	new->h.name = strdup(name);
.........................................................................
# vim: syntax=diff

